app/helpers/csv_types.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
```

Comment:
files in the `helpers` directory are intended to be helpers for views.  Everything here is static data that is used throughout the codebase.  These values should work fine as global constants.
You can create a file in the `initializers` directory and define them there.
Something like
```
CSV_TYPES_TALBES = [...] # as defined below
CSV_TYPES_REQUIRED_TABLE_NAMES = CSV_TYPES_TALBES.select { |table| table[:required?] }.map { |table| table[:klass].name }
CSV_TYPES_ALL_TABLES = CSV_TYPES_TALBES.map { |table| table[:klass] }
```
@annaswims any thoughts on the best place for this?

Reply:
I just went ahead and did this